R/seqSimVis.R

```diff
@@ -0,0 +1,98 @@
+
+
+#' \code{seqSimVis()} Calculates sequence similarity and plots dot plots based on the
+#' three highest similarity values to visualize
+#'
+#'
+#' @param gene_of_int (char) gene of interest
+#' @param system (char) system of interest
+
+#' @examples
+#' seqSimVis("BRCA1", "PHALY")
+#'
+#' @export
+seqSimVis <- function(gene_of_int, system) {
```

Comment:
Consider having the function return the similarity values for further analysis/visualization if necessary.